src/averageAges.js

```diff
@@ -14,12 +14,23 @@
  *
  * @return {number}
  */
-function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+function calculateMenAverageAge(people, century = true) {
+  const filterArr = people.filter((person) => {
+    return person.sex === 'm'
+    && (Math.ceil(person.died / 100) === century || century === true);
+  });
+
+  const ages = filterArr.map((person) => {
+    return person.died - person.born;
+  });
+
+  const getSum = (sum, currentNum) => {
+    return sum + currentNum;
+  };
+
+  const sumAge = ages.reduce(getSum, 0);
```

Comment:
you create this functions couple of times below in the code, create helper functions instead